internal/controller/datamovement_controller.go

```diff
@@ -663,7 +955,7 @@ func (r *DataMovementReconciler) getStorageNodeNames(ctx context.Context, dm *nn
 	}
 
 	if targetAllocationSetIndex == -1 {
-		return nil, newInvalidError("ost allocation set not found")
+		return nil, newInvalidError("OST allocation set not found")
```

Comment:
This reverts a commit you made a few weeks ago.

Reply:
Whoops. I think things got weird since this branch's original parent was _before_ I made the revert on the master branch. I must have missed this when I rebased the branch before making this PR. The main thing is the `"ost"` a few lines above.